lightkurve/mast.py

```diff
@@ -0,0 +1,91 @@
+"""Functions which wrap `astroquery.mast` to obtain Kepler/K2 data from MAST."""
+import numpy as np
+from astroquery.mast import Observations
+from astroquery.exceptions import ResolverError
+
+
+class ArchiveError(Exception):
+    """Raised if there is a problem accessing data."""
+    pass
+
+
+def download_products(products):
+    """Download data products from MAST.
+
+    Downloads will be cached using astroquery.mast's caching system.
+
+    Returns
+    -------
+    paths : astropy.Table.Column
+        Detailing the paths of the downloaded or cached products.
+    """
+    dl = Observations.download_products(products, mrp_only=False)
```

Comment:
Could you add a comment mention why we want ``mrp_only`` to always be ``False``

Reply:
Good thinking!!